docassemble/MAVirtualCourt/data/questions/209A_affidavit.yml

```diff
@@ -0,0 +1,507 @@
+mandatory: True
+code: |
+  interview_metadata # make sure we initialize the object
+  if not defined("interview_metadata['complaint_209A_Affidavit']"):
+    interview_metadata.initializeObject('complaint_209A_Affidavit')
+  interview_metadata['complaint_209A_Affidavit'].update({
+    'title': '209A Complaint Affidavit',
+    'short title': '209A Complaint Affidavit',
+    'description': '209A Complaint Affidavit',
+    'original_form': 'https://www.mass.gov/doc/209a-affidavit/download',
+    'allowed courts': [
+      'Boston Municipal Court',
+      'District Court',
+      'Probate and Family Court',
+      'Superior Court',
+    ],
+    'preferred court': 'Any court (no preferred court)',
+    'categories': [
+      'Domestic Violence / Safety of self',
+      'Family Law',
+    ],
+    'logic block variable': 'complaint_209A_Affidavit',
+    'attachment block variable': 'complaint_209A_Affidavit_attachment',
+    'typical role': 'plaintiff',  })
+---
+objects:
+  - incidents_of_abuse: DAList.using(minimum_number=1, object_type=DAObject, complete_attribute='complete')
+---
+features:
+  question help button: True
+---
+comment: |
+  Hopefully this screen will be unnecessary when merged in to parent
+mandatory: True
+question: |
+  Download your 209A Affidavit 
+buttons:
+  - Exit: exit
+  - Restart: restart
+attachment code: complaint_209A_Affidavit_attachment
+---
+variable name: interview_metadata['complaint_209A_Affidavit']['built_in_fields_used']
+data:
+  - variable: user_name_full
+  - variable: defendant
+  - variable: signature_date
+  - variable: user_signature
+---
+variable name: interview_metadata['complaint_209A_Affidavit']['field_list']
+data:
+  - variable: affidavit_body
+    field_type: text
+  - variable: incident_date
+    field_type: text
+---
+include:
+  - docassemble.MAVirtualCourt:basic-questions.yml
+---
+id: 209A Affidavit Intro
+continue button field: complaint_209A_Affidavit_intro
+question: |
+  209A Affidavit
+subquestion: |
+  The judge needs to understand why you need a 209A order.
+  
+  Use the Affidavit form to describe the abuse. 
+   
+  When you sign your name you are declaring, "under penalty of perjury" that what you wrote is true to the best of your knowledge.
```

Comment:
l. 68 "under penalty of perjury" and true to the best of your knowledge are both terms defined in terms part of this block. They need to be surrounded with {}s  so L. 68 should read 

When you sign your name you are declaring

"{under penalty of perjury}" that what you wrote is true

{to the best of your knowledge}.